pkg/odo/cli/component/link.go

```diff
@@ -63,6 +65,23 @@ odo link dh-postgresql-apb
 Now backend has 2 ENV variables it can use:
 DB_USER=luke
 DB_PASSWORD=secret`
+
+	linkExampleExperimental = ktemplates.Examples(`# Link the current component to the 'EtcdCluster' named 'example'
+%[1]s link EtcdCluster/example
+	`)
+
+	linkLongDescExperimental = `Link component to an operator backed service
+
+If the source component is not provided, the current active component is assumed.
+
+For example:
+
+We have created a frontend application called 'frontend' using:
+odo create nodejs frontend
+
+If you wish to connect this nodejs based component to, for example, an EtcdCluster named 'example' created from etcd Operator:
+odo link EtcdCluster/example
```

Comment:
Help message is printed as
```
Link component to an operator backed service

If the source component is not provided, the current active component is assumed.

For example:

We have created a frontend application called 'frontend' using:
odo create nodejs frontend

If you wish to connect this nodejs based component to, for example, an EtcdCluster named 'example' created from etcd Operator:
odo link EtcdCluster/example

Usage:
  odo link <service-type>/<service-name> [flags]

Examples:
  # Link the current component to the 'EtcdCluster' named 'example'
  odo link link EtcdCluster/example
```
it could be updated to, ie it could have one example section only
```
Link component to an operator backed service

If the source component is not provided, the current active component is assumed.

Usage:
  odo link <service-type>/<service-name> [flags]

Examples:
  # Link the current component to the 'EtcdCluster' named 'example'
  odo link EtcdCluster/example

  # Link a nodejs component
 create a frontend application called 'frontend' using:
 odo create nodejs frontend

 If you wish to connect this nodejs based component to an EtcdCluster named 'example' created from etcd Operator:
 odo link EtcdCluster/example
```

Reply:
Before the `Usage` and `Examples` sections, we tend to provide a working example. If we look at `odo link -h` without the Experimental mode, it has similar documentation. So, I'm trying to stick with it.

Reply:
we do not have any separate examples in `odo push` `odo create`, and this line has too many `examples` :)
```
If you wish to connect this nodejs based component to, for example, an EtcdCluster named 'example' created from etcd Operator:
odo link EtcdCluster/example
```

Reply:
> we do not have any separate examples in `odo push` `odo create`

These two commands go solo. In case of `odo link`, we have a dependency to have a component first. Hence the `odo create` example there.

> and this line has too many `examples`

Haha, agreed. Made a change there. PTAL. :)